EveripediaNetwork/ep-ui
# Timeline section implementation done

# About Page Timeline Section Redesign

_This PR implements the new design of the history (timeline) section on the About page_

## How should this be tested?

1. Navigate to about page and scroll down to history section
2. Confirm Implementation matches new design

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3056
